extensions/import/src/wizard/pages/fileConfigPage.ts

```diff
@@ -340,7 +340,14 @@ export class FileConfigPage extends ImportPage {
 		let connectionUri = await sqlops.connection.getUriForConnection(this.model.server.connectionId);
 		let queryProvider = sqlops.dataprotocol.getProvider<sqlops.QueryProvider>(this.model.server.providerName, sqlops.DataProviderType.QueryProvider);
 
-		let query = `SELECT name FROM sys.schemas`;
+		let query = '';
+		if (this.databaseDropdown.value != null)
```

Comment:
I think we're trying to use `undefined` instead of `null` generally.  Or just checking for truthiness.  So I think either `if (this.databaseDropdown.value !== undefined)` or `if (this.databaseDropdown.value)` is preferred here?